payments-core/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -831,8 +830,8 @@ class CardInputWidget @JvmOverloads constructor(
     }
 
     private fun handlePossibleCardBrandsChanged(brands: List<CardBrand>) {
-        val currentBrand = cardBrandView.brand
         cardBrandView.possibleBrands = brands
+        val currentBrand = cardBrandView.brand
```

Comment:
if this ordering is important, we should definitely have a test for the behavior it affects! This is the kind of thing that's super easy to break. (I know you said you're working on tests, just flagging that this seems like something that should be covered)

Reply:
niceee